assets/app/view/game/merge.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'view/game/actionable'
+require 'view/game/undo_and_pass'
+
+module View
+  module Game
+    class Merge < Snabberb::Component
```

Comment:
Is there any way we can get 1817 to use some of this code?

Reply:
I have not studied 1817 in detail but it would be good if it was. I don't recall such a situation in 1817, where there is a list of candidate corporations to be merged into a target, but maybe there is? 
I suppose it could be used for a Merge/Decline question of one mergee into one merger.

Reply:
In acquisitions once a player has won the bid they then choose the corporation to merge with it.

Reply:
Is there such a dialog implemented today? If not it sounds like this can be generalized to cover for that as well.  
Will try to generalize it a bit.

Reply:
Have generalized it a bit. But still not sure if it is possible to use it in 1817. Maybe in 1824?

Reply:
has this been attempted yet? that's the only thing i'd look into for this commit

Reply:
I have not done any 1817 related work. So maybe @tobymao or @jenf could try to use it for acquisitions?